shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -419,4 +414,14 @@ - (id)accessibilityContainer {
     VisitObjectsRecursivelyAndRemove(child, doomed_uids);
 }
 
+void AccessibilityBridge::HandleEvent(NSDictionary<NSString*, id>* annotatedEvent) {
+  NSString* type = annotatedEvent[@"type"];
+  if ([type isEqualToString:@"scroll"]) {
+    // TODO(tvolkert): provide meaningful string (e.g. "page 2 of 5")
+    UIAccessibilityPostNotification(UIAccessibilityPageScrolledNotification, @"");
+  } else {
+    assert(false);
```

Comment:
I am not sure this is a standard way to assert in this part of the code base

Reply:
Changed to `NSCAssert` (see other top level comment).